portal/views/views_instances.py

```diff
@@ -81,6 +81,25 @@ def view_instance(name):
                                 instance_log=instance_log)
 
 
+@app.route('/instances-log-xhr/<name>/<container>/<lines>', methods=['GET'])
```

Comment:
- function to get logs based on the instance name, container and lines sent from html
- the log query is hardcoded. I saw there is a get_instance_logs() function imported, by it gets the logs using only the instance name, so I'm keeping the hard coded version